### DISCUSSION TITLE
Data

### DISCUSSION THREAD

**Discussion by AfolabiOlaoluwa** [planetscale/discussion#217] [Q&A] [answered]
What happens to my data if I didn't get to pay for my purchased instance? Do I get the chance to pull my Database? 

I am currently using AWS. I was a bit down and not able to attend to my instance. AWS terminated my instance and I couldn't have access to anything. Not even the Database. Now I must pay before it's reinstated.

**Comment by fooforge** [comment, marked as answer]
Hey @AfolabiOlaoluwa,

If you have forgotten to pay for your database on the Scaler plan you have one month to pay the bill before we put your database [to sleep](https://docs.planetscale.com/concepts/database-sleeping). When a database is sleeping, all data is backed up, but you will not be able to connect to any branches on the database until it has been woken up. Once the outstanding invoice(s) have been paid, the database will be reawakened.

In case you want to migrate off of PlanetScale for whatever reason we will obviously give you access to your data as nobody should end up in a situation where their data is being held hostage. All that is needed in this case is a ticket opened through our [support portal](https://support.planetscale.com/), and we'll work it out.

This is also a good opportunity for a reminder to [reach out to us early](https://support.planetscale.com/) in case of any issues so that we can figure things out. We're nice people, we like to help and we want to see your businesses and projects grow along with us.

**Reply by AfolabiOlaoluwa** [reply]
That you so much. 

If I get you right, you meant that while I am being downgraded automatically back to freebie 5G, I can still download all my backup of 10G worth of data or just the freebie (5G) data?

**Reply by fooforge** [reply]
Hey @AfolabiOlaoluwa,

Turned out my information on how we're handling these cases was a bit outdated. I have updated my earlier response to reflect that. Holler if you have any follow-up questions. :-)

**Reply by AfolabiOlaoluwa** [reply]
Seen. Gracias.
